docs/locale/ja/LC_MESSAGES/use-plugin-on-sonarqube.po

```diff
@@ -0,0 +1,126 @@
+msgid ""
+msgstr ""
+"Project-Id-Version: PROJECT VERSION\n"
+"Report-Msgid-Bugs-To: EMAIL@ADDRESS\n"
+"POT-Creation-Date: 2018-07-14 14:44+0000\n"
+"PO-Revision-Date: YEAR-MO-DA HO:MI+ZONE\n"
+"Last-Translator: FULL NAME <EMAIL@ADDRESS>\n"
+"Language-Team: LANGUAGE <LL@li.org>\n"
+"MIME-Version: 1.0\n"
+"Content-Type: text/plain; charset=utf-8\n"
+"Content-Transfer-Encoding: 8bit\n"
+"Generated-By: Babel 1.3\n"
+
+#: ../../use-plugin-on-sonarqube.rst:2
+msgid "Use SpotBugs Plugin on SonarQube"
+msgstr ""
+
+#: ../../use-plugin-on-sonarqube.rst:4
+msgid ""
+"`SpotBugs SonarQube Plugin <https://github.com/spotbugs/sonar-findbugs>`_"
+" uses major SpotBugs plugin such as `fb-contrib <http://fb-"
+"contrib.sourceforge.net/>`_ and `Find Security Bugs "
+"<http://h3xstream.github.io/find-sec-bugs/>`_. However, if you want to "
+"use other SpotBugs plugin, you need to build own SonarQube plugin. For "
+"detailed requirements on SonarQube plugin, see `SonarQube official "
+"guideline <https://docs.sonarqube.org/display/DEV/Developing+a+Plugin>`_."
+msgstr ""
+
+#: ../../use-plugin-on-sonarqube.rst:7
+msgid "Create Maven Project"
+msgstr ""
+
+#: ../../use-plugin-on-sonarqube.rst:9
+msgid ""
+"Follow interaction in `SonarQube official guideline "
+"<https://docs.sonarqube.org/display/DEV/Build+Plugin#BuildPlugin-"
+"CreateaMavenProject>`_. It is recommended to use sub-module, to manage "
+"both of SpotBugs plugin and SonarQube plugin in one project. You can "
+"refer `this module <https://github.com/KengoTODA/guava-helper-for-"
+"java-8/tree/master/sonarqube-plugin>`_ as example."
+msgstr ""
+
+#: ../../use-plugin-on-sonarqube.rst:12
+msgid ""
+"You also need to configure ``sonar-packaging-maven-plugin``, to make your"
+" plugin depends on `SpotBugs SonarQube Plugin "
+"<https://github.com/spotbugs/sonar-findbugs>`_. For instance, if you're "
+"using SonarQube 6.7 LTS, your plugin requires SpotBugs SonarQube Plugin "
+"version 3.7, so configuration should be like below:"
+msgstr ""
+
+#: ../../use-plugin-on-sonarqube.rst:23
+msgid "Generate rules.xml"
+msgstr ""
+
+#: ../../use-plugin-on-sonarqube.rst:25
+msgid ""
+"SonarQube doesn't understand the Bug Pattern metadata provided for "
+"SpotBugs, so we need to convert ``findbugs.xml`` and ``messages.xml`` to "
+"SonarQube format named ``rules.xml``."
+msgstr ""
+
+#: ../../use-plugin-on-sonarqube.rst:27
+msgid ""
+"If your SpotBugs plugin isn't complex, you can simply introduce "
+"`SonarQube rule xml generator Maven Plugin <https://github.com/KengoTODA"
+"/sonarqube-rule-xml-generator>`_ to generate ``rules.xml``. Follow `the "
+"interaction described in its README <https://github.com/KengoTODA"
+"/sonarqube-rule-xml-generator#how-to-use>`_."
+msgstr ""
+
+#: ../../use-plugin-on-sonarqube.rst:30
+msgid "Update RulesDefinision.java"
+msgstr ""
+
+#: ../../use-plugin-on-sonarqube.rst:32
+msgid ""
+"Your ``SonarQubeRulesDefinition.java`` should load generated "
+"``rules.xml`` to FindBugs repository."
```

Comment:
"to your FindBugs repository"

Reply:
It's not 'your' but the repository generated by [sonar-findbugs](https://github.com/spotbugs/sonar-findbugs).
Maybe we can say 'existing' repository?

Reply:
Just "the FindBugs repository" should be fine then.